src/carbonable/erc3525/IERC3525Full.cairo

```diff
@@ -97,4 +97,20 @@ namespace IERC3525Full {
 
     func tokenOfOwnerByIndex(owner: felt, index: Uint256) -> (tokenId: Uint256) {
     }
+
+    //
+    // Slot Enumerable
+    //
+
+    func slotCount() -> (count: Uint256) {
+    }
+
+    func slotByIndex(index: Uint256) -> (slot: Uint256) {
+    }
+
+    func tokenSupplyInSlot(slot: Uint256) -> (supply: Uint256) {
```

Comment:
The returning value should not be name `totalSupply` ?

If the answer is `yes`, we should change in `library.cairo` also

Reply:
changed to total_amount to reflect EIP